### ISSUE TITLE
R640 CPU frequency set for "performance"

### ISSUE THREAD

**Issue by stephen-soltesz** [m-lab/epoxy-images#190]
R640's should be configured to use the "performance" CPU frequency governor rather than the default "powersave".

After manually setting the "performance" governor on mlab3-hkg01 using:

```
sudo apt-get install cpufrequtils
for i in `seq 0 31` ; do sudo cpufreq-set --cpu $i --governor performance ; done
```

Shortly after the very high 95th% polling interval from tcpinfo became much closer to the median.

<img width="583" alt="Screen Shot 2020-11-05 at 5 09 24 PM" src="https://user-images.githubusercontent.com/1085316/98302445-62e42980-1f8a-11eb-9d6a-9ad258fc7174.png">

And, the overall pod CPU usage and load dropped, without an apparent increase in CPU temperature.

<img width="441" alt="Screen Shot 2020-11-05 at 5 12 14 PM" src="https://user-images.githubusercontent.com/1085316/98302448-65468380-1f8a-11eb-9d9a-b9bf77b0278e.png">

**Comment by stephen-soltesz** [comment]
Talking with @nkinkade about this issue, we wondered whether the same issue may have been the explanation for why we saw a performance drop (i.e. ~60% CPU overhead) on R630s when moving from CoreOS to Ubuntu.

Running the same command on mlab3-lga05, we saw similar (though not as dramatic) improvements.

<img width="576" alt="Screen Shot 2020-11-05 at 6 17 06 PM" src="https://user-images.githubusercontent.com/1085316/98307207-3e407f80-1f93-11eb-8350-83a5211fb09d.png">
<img width="432" alt="Screen Shot 2020-11-05 at 6 17 26 PM" src="https://user-images.githubusercontent.com/1085316/98307209-3ed91600-1f93-11eb-8d38-ad787d63b33e.png">
